utils/ratlib.py

```diff
@@ -128,3 +159,73 @@ def try_parse_uuid(suspect: str) -> UUID:
 
     else:
         return result
+
+
+# color functions
+def color(text: str, text_color: Colors, bg_color=None) -> str:
+    """
+    Colorizes the given string with the specified color.
+    Args:
+        text: The text to colorize
+        text_color: a Colors.COLOR
+        bg_color: if specified, the background color
+
+    Returns:
+        str: colorized string
+    """
+    if bg_color is not None:
+        bg_color_code = bg_color.replace('\x03', '')
+        return f'{text_color},{bg_color_code}{text}{Colors.FORMAT_RESET}'
+    else:
+        return f'{text_color}{text}{Colors.FORMAT_RESET}'
+
+
+def bold(text: str) -> str:
+    """
+    Makes the text bold.
+    Args:
+        text: The text.
+
+    Returns:
+        str: the bolded text
+
+    """
+    return f'{Colors.FORMAT_BOLD}{text}{Colors.FORMAT_RESET}'
```

Comment:
resetting formatting here will have unintended consequences, such as if thi string were to then be wrapped in a color block the colour would be terminated prematurely. In other words, this method does not stack.

Demonstration:
```py
@command("debug-demo-clear")
async def cmd_dclear(context: Context):
    bold_str = bold("<bold>My boldened text</bold>")
    fstring = color(f"my beautyful {bold_str} text!", Colors.GREEN)
    await context.reply(fstring)
```
output: 
![image](https://user-images.githubusercontent.com/3110986/42837246-5da15eb8-89b2-11e8-9a48-dce94a4b1fbd.png)
expected: the whole output string is in green.

Root cause: `colors.FORMAT_RESET` at the end of your formatting methods.

I believe the proper way to toggle a formatter is to repeat the format char without arguments.

for bold you would wrap text in 
```py
f"{Colors.FORMAT_BOLD}text{Colors.FORMAT_BOLD}"
```